dapp/src/hooks/useReferral.ts

```diff
@@ -17,10 +18,17 @@ export default function useReferral(): UseReferralReturn {
     env.REFERRAL,
   )
   const { openModal } = useModal()
+  const { isEmbed } = useIsEmbed()
 
   const detectReferral = useCallback(() => {
     const param = referralProgram.getReferralFromURL()
 
+    if (isEmbed) {
+      // TODO: Set correct referral for embedded dApp
+      setReferral(0)
```

Comment:
The `embed` parameter is not boolean. We will use it to pass the partner integration ID what will have to map to a referral ID. For Ledger Live we will have `embed=ledger-live` and we need to map it to the referral ID assigned for Ledger Live, which is number `2083` (source: [Coda page](https://coda.io/d/Acre_d3Vs1FswTQA/Referrals_sudl9wjh#_lubtctD5))